playwright.config.ts

```diff
@@ -8,35 +8,29 @@ export default defineConfig({
   testDir: "playwright/tests",
   fullyParallel: true,
   forbidOnly: !!process.env.CI,
-  // TODO hardcoded values should be extracted to ENVs
   retries: process.env.CI ? 2 : 0,
-  workers: process.env.CI ? 2 : 2,
+  workers: process.env.CI ? 4 : 2,
```

Comment:
@saleor/qa-guild I suggest moving this config options to vars that can be configurable from the repository settings (just like secrets) instead of hardcoding them

Reply:
@andrzejewsky I've changed it to:

``
  retries: env.CI ? parseInt(env.CI_PW_RETRIES) : parseInt(env.LOCAL_PW_RETRIES),
  workers: env.CI ? env.CI_PW_WORKERS:env.LOCAL_PW_WORKERS,
``

is it what you meant?